sw/tools/qtc.creator.user_template

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE QtCreatorProject>
-<!-- Written by QtCreator 4.1.0, 2017-01-23T23:23:44. -->
+<!-- Written by QtCreator 4.4.0, 2017-09-08T21:31:25. -->
```

Comment:
this file should probably be a part of .gitignore

Reply:
Actually, it should not. The template is used to generate an accepted qtc project.user. Unfortunately it currently means that the template has to be updated when qtc changes. Looking for a better solution, but nothing as of yet.
(other than that, I can recommend qt over eclipse. It only shows you the files that are actually used)